README.md

```diff
@@ -173,7 +173,8 @@ to add this to your `proguard-rules.pro`:
 
 ```proguard
 -keepattributes *Annotation*, InnerClasses
--dontnote kotlinx.serialization.SerializationKt
+-dontnote kotlinx.serialization.AnnotationsKt # core serialization annotations
+-keep,includedescriptorclasses class kotlinx.serialization.json.**$$serializer { *; } # JsonElement serializers
```

Comment:
With the goal of keeping `core` format agnostic, would it make sense to move this `json`-specific rule to its own section at the end of the `core` rules?

Reply:
Makes sense. @qwwdfsad You can also add commentary like 'you can omit this line if you aren't using Json format'